### DISCUSSION TITLE
related to python script file

### DISCUSSION THREAD

**Discussion by priya23595** [Sunbird-cQube/community#797] [Q&A] [answered]
How we use add_nifi_template.py python script file in cqube  what it exactly outcome ?
how python is usefull in cqube workflow, exact working and output of python script file ?

**Comment by DevashishRaghav** [comment]
@padmanabha1234

**Comment by priya23595** [comment]
hi
can u provide email of this github id @padmanabha1234
<https://github.com/padmanabha1234> so that i can ask question on mail

**Comment by padmanabha1234** [comment, marked as answer]
Hi @priya23595 
The use of add_nifi_template.py is it will create the desired processor groups in the nifi according to the storage types.
in order to run this file. need to create and fill the config file.
        [CREDs]
        storage_type = oracle/aws/local/azure
        nifi_host = http://localhost
        nifi_port = 8080
        state_name = JH
        s3_access_key = 
        s3_secret_key = 
        s3_input_bucket = 
        processing_time = * * * * * ?
        plugin_time = * * * * * ?
        adapter_schedule_time = * * * * * ?
        azure_container_name = 
        azure_account_name = 
        azure_account_key = 
        minio_end_point = 
        minio_port = 
        minio_bucket = 
        minio_access_key = 
        minio_secret_key = 
        instance_type = VSK

where it will move the files from storage process_input folder to the code. and it will execute the **yarn cli commands**

**Comment by padmanabha1234** [comment]
Hi @priya23595 
Contact me if you have any queries. 
**padmanabha.j@tibilsolutions.com**
